src/env.spec.ts

```diff
@@ -85,6 +85,32 @@ describe("getConfig", () => {
       site: "datadoghq.com",
       enableXrayTracing: false,
       enableDDTracing: true,
+      addExtension: false,
+      enableTags: true,
+      injectLogContext: true,
+      exclude: [],
+    });
+  });
+
+  it("gets a mixed configuration when addExtension is set", () => {
```

Comment:
What do you mean here by "gets a mixed configuration"?

Reply:
I took the "gets a mixed configuration" from the test above but from what I understand is that when the customer provides actual parameters under the custom section then we get a config that is mixed between defaults and stuff that the customer explicitly defined. 
So its not entirely the default or custom but its a mix between the both.

Reply:
Hmm I think "mixed configuration" is a bit confusing, I would consider rewording this test case. What, in your words, is this test case trying to test?